Source/WPEFramework/PluginServer.h

```diff
@@ -2978,7 +2978,8 @@ namespace PluginHost {
         {
             _dispatcher.Revoke(job);
         }
-        inline const PluginHost::Config& Configuration() const
```

Comment:
I would not remove the const version. I understand you now need a non const version but perhaps somebody in the future might have a const pluginhist and wishes to read the config, then he would not be able to do so (without const cast, but I think we agree that would not be a good solution, So I would leave the const version as well.